pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -184,22 +196,22 @@ func CreateCustomResourceDefinitions(clientset apiextensionsclient.Interface) er
 				}
 			case apiextensionsv1beta1.NamesAccepted:
 				if cond.Status == apiextensionsv1beta1.ConditionFalse {
-					log.Errorf("Name conflict: %s", cond.Reason)
+					scopedLog.WithError(fmt.Errorf(cond.Reason)).Error("Name conflict for CRD")
```

Comment:
Is this safe? K8s can trick us into any format string. Use `errors.New()`?

Reply:
good point. I will change it so it isn't a format string.